model/project_aliases_test.go

```diff
@@ -161,71 +159,71 @@ func (s *ProjectAliasSuite) TestFindAliasInProject() {
 	s.Len(found, 2)
 }
 
-func (s *ProjectAliasSuite) TestMergeAliasesWithParserProject() {
-	s.Require().NoError(db.ClearCollections(ProjectAliasCollection, ParserProjectCollection, VersionCollection))
-	v1 := Version{
-		Id:         "project-1",
-		Identifier: "project-1",
-		Requester:  evergreen.RepotrackerVersionRequester,
-	}
-	a1 := ProjectAlias{
-		ProjectID: "project-1",
-		Alias:     evergreen.CommitQueueAlias,
-	}
-	a2 := ProjectAlias{
-		ProjectID: "project-1",
-		Alias:     evergreen.CommitQueueAlias,
-	}
-	a3 := ProjectAlias{
-		ProjectID: "project-1",
-		Alias:     evergreen.GithubPRAlias,
-	}
-	s.NoError(a1.Upsert())
-	s.NoError(a2.Upsert())
-	s.NoError(a3.Upsert())
-	s.NoError(v1.Insert())
-
-	parserProject := &ParserProject{
-		Id: "project-1",
-		PatchAliases: []ProjectAlias{
-			{
-				ID:        mgobson.NewObjectId(),
-				ProjectID: "project-1",
-				Alias:     "alias-2",
-			},
-			{
-				ID:        mgobson.NewObjectId(),
-				ProjectID: "project-1",
-				Alias:     "alias-1",
-			},
-		},
-		CommitQueueAliases: []ProjectAlias{
-			{
-				ID:        mgobson.NewObjectId(),
-				ProjectID: "project-1",
-			},
-		},
-	}
-	s.NoError(parserProject.TryUpsert())
-
-	projectAliases, err := FindAliasInProjectOrRepo("project-1", evergreen.CommitQueueAlias)
-	s.NoError(err)
-	s.Len(projectAliases, 1)
-
-	projectAliases, err = FindAliasInProjectOrRepo("project-1", evergreen.GithubPRAlias)
-	s.NoError(err)
-	s.Len(projectAliases, 1)
-	projectAliases, err = FindAliasInProjectOrRepo("project-1", "alias-1")
-	s.NoError(err)
-	s.Len(projectAliases, 1)
-	projectAliases, err = FindAliasInProjectOrRepo("project-1", "alias-2")
-	s.NoError(err)
-	s.Len(projectAliases, 1)
-
-	projectAliases, err = FindAliasInProjectOrRepo("project-1", "nonexistent")
-	s.NoError(err)
-	s.Len(projectAliases, 0)
-}
+//func (s *ProjectAliasSuite) TestMergeAliasesWithParserProject() {
```

Comment:
Is this still commented out because of the error with creating a new collection? Did you try 
```
    _ = env.DB().RunCommand(nil, map[string]string{"create": evergreen.ProjectConfigCollection})
 ```

Reply:
Works, thanks! I don't know why only some tests were giving me trouble with this and not all of them across the board.